openmc/__init__.py

```diff
@@ -39,4 +40,5 @@
 from openmc.model import Model
 
 
-__version__ = '0.14.1-dev'
+
+__version__ = importlib.metadata.version("openmc")
```

Comment:
I would recommend using dynamic metadata instead to keep this more consistent with what people expect: [dynamic metadata](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata).

Reply:
Also relevant blog post on `setup.py` deprecation: https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html